db/schema.rb

```diff
@@ -1705,6 +1705,7 @@
     t.datetime "confirmation_sent_at"
     t.bigint "school_group_id"
     t.string "unlock_token"
+    t.string "preferred_locale", default: "en", null: false
```

Comment:
Wondering if this actually belongs with a different PR..?

Reply:
oops yep.  will remove